src/map/entities/mobentity.cpp

```diff
@@ -642,6 +642,10 @@ void CMobEntity::OnMobSkillFinished(CMobSkillState& state, action_t& action)
     {
         findFlags |= FINDFLAGS_PET;
     }
+    if ((PSkill->getValidTargets() & TARGET_IGNORE_BATTLEID) == TARGET_IGNORE_BATTLEID)
```

Comment:
If this isn't related to the if-statement above it, please separate them with a newline